apps/analysis_framework/serializers.py

```diff
@@ -182,6 +187,24 @@ class Meta:
         model = AnalysisFramework
         fields = ('__all__')
 
+    def get_projects(self, obj):
+        user = None
+        if 'request' in self.context:
+            user = self.context['request'].user
+        projects = obj.project_set.exclude(Q(is_private=True) & ~Q(members=user))
+        return SimpleProjectSerializer(projects, many=True, read_only=True).data
+
+    def get_can_add_users(self, obj):
```

Comment:
better to have a different name. The name sounds like it has boolean value.

Reply:
@bewakes Let me know if you have suggestions.

`users_with_can_add_permission` sounds good?

Reply:
Maybe `users_with_add_permission`?

Reply:
Sounds good.

Reply:
Changes pushed https://github.com/the-deep/server/pull/470/commits/643fe281782d0cec95d2c7b114ed1734bd22be78